data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/model/operations/CreateEnvironmentRequest.java

```diff
@@ -12,8 +12,9 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-package com.amazonaws.blox.dataservicemodel.v1.model;
+package com.amazonaws.blox.dataservicemodel.v1.model.operations;
```

Comment:
This isn't an operation, it's a model type

Reply:
Hm. Any better categorizations for the types that only serve as input/output wrappers? I don't really think of those as `model` types, and it makes it hard to see the real domain types.

Reply:
You can name it something else but something that signifies a type and not a method/operation

Reply:
Okay, how about `wrappers`?

Reply:
sure